### PR TITLE
[ui] Add logging messages

### PR DESCRIPTION
**Pull request by evamillan**
Logs a message to the console when the following operations are performed, including the values that were changed:
- Add identity
- Move identity
- Update profile
- Enroll in organization
- Update enrollment
- Remove enrollment
- Merge identities
- Split identities
- Delete identity
- Add organization
- Add domain
- Remove domain
- Log in/out

The logs are not shown in production mode (when running `yarn serve --mode production`, for example).

### CODE REVIEW

File: ui/src/App.vue [hunk 1]
```diff
@@ -51,6 +51,7 @@ export default {
       Cookies.remove("sh_authtoken");
       Cookies.remove("sh_user");
       this.$router.push("/login");
+      this.$logger.log(`Log out user ${this.user}`);
```
Comment (sduenas) [inline ui/src/App.vue:54]: Set is as `info`.

File: ui/src/components/IndividualsTable.vue [hunk 2]
```diff
@@ -359,6 +359,10 @@ export default {
         individuals.map(individual => this.deleteItem(individual.uuid))
       );
       if (response) {
+        this.$logger.log(
```
Comment (sduenas) [inline ui/src/components/IndividualsTable.vue:362]: Set it as `debug` message.

File: ui/src/plugins/logger.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+export default {
+  install(Vue) {
+    function log(type, message, ...args) {
+      if (process.env.NODE_ENV === "production") {
+        return;
```
Comment (sduenas) [inline ui/src/plugins/logger.js:5]: I think for production we might want all messages but `debug` ones. If the app is reporting an error, users can give us useful information.

File: ui/src/plugins/logger.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+      }
+      console[type](message, ...args);
+    }
+
+    Vue.prototype.$logger = {
```
Comment (sduenas) [inline ui/src/plugins/logger.js:10]: We need some unit tests for this. For example, they can test the message is correctly written or that the log level is correctly set depending on the function called.

**Review by sduenas — CHANGES_REQUESTED** [review]
In overall the PR looks good to me but it needs some changes. For example, I think all `log` messages should be set to `debug` (I added some comments about it). Also, please rebase the PR because is failing due `flake8`checks that we already fixed in a previous commit.

**Comment by evamillan** [comment]
I updated the PR adding tests and replacing the `log` messages with `debug`.

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**